rules-tests/DowngradePhp80/Rector/Class_/DowngradePropertyPromotionRector/Fixture/with_attributes.php.inc

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Rector\Tests\DowngradePhp80\Rector\Class_\DowngradePropertyPromotionRector\Fixture;
+
+class WithAttributes
+{
+    public function __construct(
+        #[\Deprecated]
+        private int $property = 0,
```

Comment:
What about case like this?

```php
#[\Deprecated] private int $property = 0,
```

Reply:
That will be processed as well https://github.com/rectorphp/rector-src/pull/410/commits/bf8008da8f6124b04add4d67daf4f08c9ebb1413